tester_app/client/webpack.client.rails.config.js

```diff
@@ -0,0 +1,45 @@
+// Run like this:
+// cd client && npm run build:dev
+// Note that Foreman (Procfile.dev) has also been configured to take care of this.
+
+// NOTE: All style sheets handled by the asset pipeline in rails
+
+const webpack = require('webpack');
+const config = require('./webpack.client.base.config');
+
+const devBuild = process.env.NODE_ENV !== 'production';
```

Comment:
> Are you suggesting we should have all those available?

No, it's just an example how to define global, not what we should define. We should add globals only if we need them somewhere.